model-optimizer/mo/graph/graph.py

```diff
@@ -975,9 +975,6 @@ def clean_up(self, undead_node_types: list = None):
         if undead_node_types is None:
             undead_node_types = []
 
-        if 'fw' in self.graph and self.graph['fw'] == 'tf':
```

Comment:
How is this change connected with v7 removal? Is it really not needed?

Reply:
The TensorFlow offload was used in two cases: when MO falls back to TF to infer the shape of unsupported operation and when we dump part of the TF model to the IR to infer it using special custom layer. The latter feature has been removed a couple of releases ago and this is a leftover from it. This change does not affect the TF fallback but should result in some small performance improvement in clean_up function.